test/built-ins/Array/prototype/concat/array-concat-arity.js

```diff
@@ -0,0 +1,11 @@
+// Copyright 2014 the V8 project authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// Flags: --harmony-arrays --harmony-classes
```

Comment:
Or should they be converted to "features" front matter, or are we dropping that idea @bterlson ?